src/main/java/seedu/bankwithus/Parser.java

```diff
@@ -44,9 +44,9 @@ public Parser(AccountList accountList) {
      */
     public void parseUserInput(String input) throws CommandNotFoundException, IOException {
         // Split input by space
-        String[] split = input.trim().split("\\s+", 2);
+        String[] split = input.trim().split("\\s+", 3);
         String command = split[0];
-        String args = split.length == 2 ? split[1] : "";
+        String args = split.length > 1 ? split[1] : "";
```

Comment:
No do not change this. I am willing to accept a lot of things, but this is not one of them. If you wish to accept more than 1 argument, then use the split function again in the specific command case to split args